src/Tests/Microsoft.Diagnostics.Monitoring.StartupHook.UnitTests/BackgroundService/BackgroundServiceTests.cs

```diff
@@ -0,0 +1,129 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Diagnostics.Monitoring.TestCommon;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.Diagnostics.Monitoring.StartupHook
+{
+    [TargetFrameworkMonikerTrait(TargetFrameworkMonikerExtensions.CurrentTargetFrameworkMoniker)]
+    public sealed class BackgroundServiceTests
+    {
+        [Fact]
+        public void ConstructionWorks()
+        {
+            using BackgroundService _ = new MockBackgroundService();
+        }
+
+        [Fact]
+        public async Task StartBackgroundTask()
```

Comment:
nit:

For test names, let's separate the name of the method from what aspect of it is being tested.

e.g.

- `Start_StartsBackgroundTask`
- `Stop_TriggersCancellation`
- ...